pytest-selenium/regions/my_highlights.py

```diff
@@ -434,7 +437,7 @@ class ChapterFilters(FilterSelection):
             @property
             def chapters(self) \
                     -> List[MyHighlights.FilterBar.ChapterFilters.Chapter]:
-                r"""Access the book chapter filters.
+                """Access the book chapter filters.
```

Comment:
Is Black complaining about the raw string? The doctext should be raw because it has a line continuation in the `rtype`.

Reply:
No. I just did not knew what raw string was